Sources/PluginLibrary/Descriptor.swift

```diff
@@ -285,6 +286,9 @@ public final class EnumValueDescriptor {
     self.index = index
     self.enumType = enumType
     aliasOf = aliasing
+
+    let fullName = "\(enumType.fullName).\(proto.name)"
```

Comment:
Does this work better as a `lazy` property (or do you still have performance concerns there)?